README.md

```diff
@@ -9,19 +9,95 @@ provisioning teams on a deployed [Concourse CI](https://concourse.ci/)
 instance.
 
 
-## Deploying
+## Setup
 
 #### Step One: Create a client in UAA for this app
 
 This application uses oauth to perform actions on your behalf in UAA.  To add a new oauth client in UAA, run the following command:
 
 	uaac client add [your-client-id] --name "Concourse CI Broker" --scope "cloud_controller.read" --authorized_grant_types "authorization_code,client_credentials,refresh_token" --authorities "cloud_controller.admin" --autoapprove "true" --redirect_uri [url-for-concourse-ci]/auth/uaa/callback -s [your-client-secret]
 
-Remember the client-id and client-secret, you'll need them in the next step
+**Remember the client-id and client-secret, you'll need them for Deployment**
 
-**TODO**
+## Deployment
+
+### Automated
+
+The easiest/recommended way to deploy the broker is via the [Concourse](http://concourse.ci/) pipeline.
+
+1. Create a `ci/credentials.yml` file, and fill in the templated values from [the pipeline](ci/pipeline.yml).
+1. Deploy the pipeline.
+
+    ```bash
+    fly -t lite set-pipeline -n -c ci/pipeline.yml -p deploy-concourse-broker -l ci/credentials.yml
+    ```
+
+### Manual
+
+1. Clone this repository, and `cd` into it.
+1. Target the space you want to deploy the broker to.
+
+    ```bash
+    $ cf target -o <org> -s <space>
+    ```
+
+1. Set the `environment_variables` listed in [the 'broker-deploy-production' step of the 'push-concourse-broker-production' job](ci/pipeline.yml).
```

Comment:
I think we should probably take the same approach the uaa-extras app does this, and just put the needed env vars in the `manifest.yml`, but commented, and instruct foks to just uncomment and fill out.

See: https://github.com/18F/cg-uaa-extras/blob/staging/manifest.yml#L16